brix-core/src/main/java/org/brixcms/jcr/RepositoryUtil.java

```diff
@@ -29,6 +29,33 @@
 public class RepositoryUtil {
     private static final Logger logger = LoggerFactory.getLogger(Brix.class);
 
+    /**
+     * register a brix basic node as a unstructured node type, reason is: accoring to JCR spec 2,
+     * a mixin may only have other mixins as supertypes
+     *
+     * @param workspace
+     */
+    public static void registerBrixUnstructuredMixin(Workspace workspace) {
+        String cnd = "[brix:unstructured] mixin" +
+                "  - * (undefined) multiple " +
+                "  - * (undefined) " +
+                "  + * (nt:base) sns version ";
+
+        NodeTypeManager manager = null;
+        try {
+            manager = workspace.getNodeTypeManager();
+
+            if (manager.hasNodeType("brix:unstructured") == false) {
+                CndImporter.registerNodeTypes(new StringReader(cnd), workspace.getSession());
+            }
+
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
ah ok - it wont fail in this context exactly here but spill out logdata but at a later point when the jcr repo wants to go up then the repo will throw an additional error and finally kill the startup of the application; You are right that a big error here may be better - I just noticed in the past that additional log chatter and a big jcr exception often helps me to get faster to a fix than an too early stop of it but I dont mind it;